rustls/src/verifybench.rs

```diff
@@ -55,7 +54,7 @@ fn test_reddit_cert() {
         || (),
         |_| {
             let dns_name = webpki::DNSNameRef::try_from_ascii_str("reddit.com").unwrap();
-            V.verify_server_cert(&anchors, &chain[..], dns_name, &[])
+            V.verify_server_cert(&anchors, &chain[..], dns_name, &[], fixed_time())
```

Comment:
`fixed_time()` should be called before `bench()` so that its execution time isn't included in the measurement.

Reply:
Fixed.